docker-compose.env.yaml.example

```diff
@@ -0,0 +1,5 @@
+DEBUG=1
+APPLY_MIGRATIONS=1
+CREATE_SUPERUSER=1
+
+TOKEN_AUTH_AUTHSERVER_URL=http://tunnistamo-backend:8000/openid
```

Comment:
Optional: Could document these variables, especially as more are added.

Reply:
Added documentation to the README.